### PR TITLE
Dependency & import cleanup

### PR DESCRIPTION
**Pull request by chrieke**
- Makes Jupyter installation optional (removed from req, recommended in docs)
- Simplify Ipython kernel check, now just returns the map object if not run in ipython
- Minor improvements to required imports

### CODE REVIEW

**Comment by nekhtiari** [comment]
LGTM

File: up42/tools.py [hunk 1]
```diff
@@ -4,16 +4,24 @@
...
+from .utils import get_logger, folium_base_map, DrawFoliumOverride
+
+try:
+    from IPython.display import display
+    from IPython import get_ipython
+
+    get_ipython().run_line_magic("matplotlib", "inline")
+except (ImportError, AttributeError):
+    # No Ipython installed, Installed but run in shell
+    pass
```
Comment (rodrigoalmeida94) [inline up42/tools.py:24]: Nice!

File: up42/utils.py [hunk 2]
```diff
@@ -5,14 +5,13 @@
 import tempfile
 import tarfile
 
-import folium.plugins
-import geojson
-import geopandas as gpd
+import folium
+from folium.plugins import Draw
```
Comment (rodrigoalmeida94) [inline up42/utils.py:9]: Is this in requirements? or is it installed by default?

File: docs/installation.md [hunk 3]
```diff
@@ -14,6 +14,14 @@ If you have an existing installation, update to the newest version via:
 pip install up42-py --upgrade
 ```
 
+!!! Info "Optional: Jupyter Lab"
+    Alltough optional, the UP42 Python SDK is optimally used in a Jupyter notebook,
```
Comment (markusuwe) [inline docs/installation.md:18]: Although

File: up42/utils.py (hunk 2 above)
Comment (chrieke) [reply up42/utils.py:9]: default

**Pull request merged**